File: lib/features/presentation/models/reminders_model.dart
class Reminders {
      String? date;
      String? firebaseUid;
      List<String>? userReminders;

  Reminders({
        this.date,
        this.firebaseUid,
        this.userReminders,
  });

  factory Reminders.fromJson(Map<String, dynamic> json) {
    return Reminders(
      date: json['date'] as String,
      firebaseUid: json['firebase_uid'] as String,
      userReminders: json['user_reminders'] != null
        ? List<String>.from(json['user_reminders'] as List<dynamic>) 
        : null,
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'date': date,
      'firebase_uid': firebaseUid,
      'user_reminders': userReminders,
    };
  }
}

File: lib/backend/app/server/models/reminders_model.py
from beanie import Document
from pydantic import EmailStr
from typing import Dict, List, Optional
from datetime import datetime
class Reminders(Document):
    class Settings:
        name = "Reminders"
    date : str  # Date of the day
    firebase_uid : str
    user_reminders : List[str] # List of user reminders


    # can have all() method that calls up all of the reminders (but for what purpose?)

